source/puppet/4.9/style_guide.markdown

```diff
@@ -398,7 +398,7 @@ file {
     * => $defaults ;
 
   '/tmp/foo' :
-   content => "foos content"
+   content => 'foos content'
```

Comment:
This `content` line should end with a trailing comma as per the trailing-comma must.